docs/ci.md

```diff
@@ -0,0 +1,68 @@
+# CI for InstructLab SDG
+
+Before running any testing locally, ensure you have run `pip install -r requirements-dev.txt` in your environment.
+
+## Unit tests
+
+Unit tests are designed to test specific SDG components or features in isolation. Generally, new code should be adding or modifying unit tests.
+
+All unit tests currently live in the `tests/unit` directory and are run with [pytest](https://docs.pytest.org/) via [tox](https://tox.wiki/).
+
+To run the unit tests, you can run `tox -e unit` or `tox -e unitcov` if you want to generate coverage metrics as well.
```

Comment:
Unrlated to this PR but not sure why this isn't working for me
> ERROR: No matching distribution found for submodlib-py

Reply:
I think I see the problem submodlib.py doesn't support python 3.13 , the default on my laptop
it works when I run "tox -e py311-unit"